en/includes/guides/authentication/configure-token-exchange.md

```diff
@@ -96,6 +95,68 @@ To enable token exchange in your application:
 
 4. Click **Update** to save the configurations.
 
+{% if product_name == "Asgardeo" %}
+
+## Configure token exchange for a local user
+
+{{ product_name }} can exchange a third-party token with a token issued for an existing local user account. This is beneficial if you wish to check for blocked/disabled user accounts or to enforce Role-Based Access Control (RBAC).
+
+You can use the following properties to customize how token exchange occurs for identities with local user accounts.
+
+### Prioritize local account attributes
+
+When this configuration is enabled, {{ product_name }} includes the local user profile information in the exchanged token if the federated identity has a linked local user account. Otherwise, {{ product_name }} returns the profile information received directly from the federated identity.
+
+To prioritize linked local account attributes:
+
+1. On the {{ product_name }} console, go to **Applications**.
+
+2. Open your application from the list and go to its **User Attributes** tab.
+
+3. Scroll down and under **Attribute Resolution for Linked Accounts**, select **Use linked local account attributes**.
+
+    !!! note
+        Select **Require linked local account** for {{ product_name }} to return an error if no user account linked to the federated identity is found.
+
+### Implicit account linking
+
+You can use implicit account linking capability in the registered trusted token issuer to automatically create an account link between a local user account in {{ product_name }} and a federated identity during token exchange.
+
+You can configure lookup attributes that will be used to search for a matching local user account. If an account is found, {{ product_name }} links the local user account automatically to the federated identity.
+
+Once account links are established, administrators cannot delete them. Users can manage their own accounts links using the <a href="{{base_path}}/guides/user-self-service/manage-linked-accounts">Manage linked accounts</a> capability in the Self-service portal.
+
+!!! note
+    If prioritize local account attributes is disabled, {{ product_name }} will not perform implicit account linking even if the option is enabled.
```

Comment:
the name should be updated here as well

Reply:
Fixed with https://github.com/wso2/docs-is/pull/5401/commits/f25b5386a82be65596d876c78c7a7ecfff1848d9